### ISSUE TITLE
Support for pruning kernels?

### ISSUE THREAD

**Issue by iamweiweishi** [tensorflow/model-optimization#645]
I am using custom layers instead of Keras Layers/Models, so is it possible to prune the kernels/weights separately like 'Model pruning' in tf1.X.

conv = tf.nn.conv2d(images, pruning.apply_mask(weights), stride, padding)

**Comment by daverim** [comment]
If you are using a custom keras layer, you could switch to `conv = prune_low_magnitude(tf.keras.layers.Conv2D(weights, strides=stride, padding=padding)(images))`,

Otherwise we don't have any support for tf1 layers.
